crimsobot/cogs/cringo.py

```diff
@@ -193,30 +186,23 @@ async def cringo_main(self, ctx: CrimsoContext, card_size: int = 4) -> None:
         if ctx.guild and ctx.guild.id == 552650672965943296:
             nerf = (2 - SERVER_BONUS) * nerf
 
-        score_string, winner = await cringo.cringo_scoreboard(list_of_players)
+        game_is_cursed = name_prefix == 'None'
```

Comment:
would rather have this be at the top instead of comparing strings...
i.e. `game_is_cursed` is the source of truth on whether we're doing NoneCRINGO